content/posts/aws-autoscaling-groups-with-terraform.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Updating AWS autoscaling groups with Terraform and preserve desired capacity"
```

Comment:
Grammar: Either "updating" and "preserving", or "update" and "preserve".

Reply:
Fixed